jfix-stdlib-ratelimiter/src/main/kotlin/ru/fix/stdlib/ratelimiter/DispatcherCommonPoolScope.kt

```diff
@@ -0,0 +1,20 @@
+package ru.fix.stdlib.ratelimiter
+
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineName
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import org.slf4j.LoggerFactory
+import java.util.concurrent.ForkJoinPool
+import kotlin.coroutines.EmptyCoroutineContext
+
+object DispatcherCommonPoolScope : CoroutineScope {
```

Comment:
I think ForkJoin is not applicable to all kinds of kotlin coroutines based applications (due to presence of more coroutine-friendly `Dispatchers.Default`). That is why I recommend to do not increase complexity of jfix-stdlib-ratelimiter module by providing some CoroutineScope implementation. For tests you can use `Dispatchers.Default`.